spec/units/derivatives_spec.rb

```diff
@@ -30,4 +30,30 @@ class CustomFile < ActiveFedora::Base
       end
     end
   end
+
+  [:ffmpeg_path, :libreoffice_path, :temp_file_base, :fits_path, :kdu_compress_path,
```

Comment:
I'm seeing that these method names mirror method names at https://github.com/projecthydra/hydra-derivatives/blob/config_spec_test_refactor/lib/hydra/derivatives.rb#L49

Could these be extracted to a constant?